sample_data/sample_config.ini

```diff
@@ -180,6 +180,10 @@ dropout_ratio=0.3
 ; Maximum number of batch data to be trained.
 max_iteration=10000
 
+; Training will be stopped if we cannot get the lowest dev ppl. for this hours.
+; If this is set to 0, we don't stop training with this criteria.
+max_waiting_hour=24
```

Comment:
this method could not stop the trainer as soon as the dev-loss goes worse. Maybe it is better to remove this behavior and to recommend to set some large value (e.g. 114514) to disable early stopping virtually.